proto/ptypes/spdk.proto

```diff
@@ -9,14 +9,14 @@ service SPDKService {
     rpc ReplicaDelete(ReplicaDeleteRequest) returns (google.protobuf.Empty);
     rpc ReplicaGet(ReplicaGetRequest) returns (Replica);
     rpc ReplicaSnapshotCreate(SnapshotRequest) returns (Replica);
-    rpc ReplicaSnapshotDelete(SnapshotRequest) returns (google.protobuf.Empty);
```

Comment:
For the deletion, there is no need to or we may be not able to return the object...
Same below

Reply:
Got it. Return `&empty. Empty{}` instead. Thank you.